pact/coin-contract/v3/coin-v3.pact

```diff
@@ -456,14 +456,25 @@
           }))
       ))
 
-  (defun enforce-chainweb-account:bool (account:string guard:guard)
-    @doc "Enforce chainweb single-account naming protocol"
-    (if (and (= "k:" (take 2 account))
-             (= 66 (length account)))
-      (enforce (= (format "{}" [guard])
-                  (format "KeySet {keys: [{}],pred: keys-all}" [(drop 2 account)]))
-               "Single-key account name violation")
-      true))
+  (defun check-reserved:string (account:string)
+    " Checks ACCOUNT for reserved name and returns type if \
+    \ found or empty string. Reserved names start with a \
+    \ single char and colon, e.g. 'c:foo', which would return 'c' as type."
+    (let ((pfx (take 2 account)))
+      (if (= ":" (take -1 pfx)) (take 1 pfx) "")))
```

Comment:
This would give us 62 potential reserved names: [a-zA-Z0-9]. Is this all we're ever going to want?